code/drasil-data/Data/Drasil/Concepts/Math.hs

```diff
@@ -12,7 +12,7 @@ mathcon :: [ConceptChunk]
 
 mathcon = [angle, area, axis, calculation, cartesian, centre, change, constraint, diameter,
   equation, euclidN, euclidSpace, gradient, graph, law, line, matrix, norm, normal,
-  normalV, number, orient, parameter, perp, perpV, pi_, point, probability, rOfChng,
+  normalV, number, orient, parameter, perp, perpV, pi_, qdNegInf, qdPosInf, point, probability, rOfChng,
```

Comment:
please omit the `qd` prefix.  Just `negInf` and `posInf`.

Reply:
<img width="1440" alt="Screenshot 2021-05-13 at 7 38 51 AM" src="https://user-images.githubusercontent.com/72299644/118120746-5cfa5580-b3be-11eb-8560-667cafeb7842.png">
there's a warning that there's a parse error with NegInf, after i take out the "qd"?

Reply:
`negInf`, not `NegInf`. Look at the capitalization. Identifiers in Haskell cannot start with a capital letter, those are reserved for constructors.